scss/dialogs/_settings.scss

```diff
@@ -151,6 +152,18 @@
   }
 }
 
+.SettingsIconButton {
+  display: flex;
+  padding: 0px 20px;
+
+  .Icon {
+    width: 24px;
+    height: 100%;
+    -webkit-mask-size: 80%;
+    background-color: var(--scrollbarThumb);
```

Comment:
I would prefer a new css theming var for this. `scrollbarThumb` is very far from what this does.